projects/github-actions/repo-gardening/src/tasks/add-labels/index.js

```diff
@@ -204,7 +204,7 @@ async function getLabelsToAdd( octokit, owner, repo, number, isDraft ) {
 
 		// Boost Critical CSS.
 		const boostModules = file.match(
-			/^projects\/plugins\/boost\/app\/features\/(?<boostModule>[^/]*)\//
+			/^projects\/plugins\/boost\/app\/modules\/(?<boostModule>[^/]*)\//
```

Comment:
That looks good, thanks for making the changes. You'll need a changelog to go along those changes though.

If the features directory is to stay for some of the things (I see we still have some things in there), another option could be to support both features and modules?

```suggestion
			/^projects\/plugins\/boost\/app\/(?:modules|features)\/(?<boostModule>[^/]*)\//
```